bundle/manifests/dex-operator.clusterserviceversion.yaml

```diff
@@ -84,7 +84,7 @@ metadata:
     capabilities: Basic Install
     operators.operatorframework.io/builder: operator-sdk-v1.9.0
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
-  name: dex-operator.v0.0.4
+  name: dex-operator.v0.0.5
```

Comment:
Should these files be deleted and gitignored since they are generated and will be different for each developer?

Reply:
can't ignore this file. This version is currently ignored by the official prow build. I shoudn't have modified this version in the PR